spec/api-deprecations-spec.js

```diff
@@ -80,8 +62,8 @@ describe('deprecations', () => {
     const o = {}
 
     expect(() => {
-      deprecate.removeProperty(o, 'iDontExist')
-    }).to.throw(/Cannot deprecate a property on an object which does not have that property/)
+      deprecate.removeProperty(o, 'iDoNotExist')
+    }).to.throw(/iDoNotExist/)
```

Comment:
We likely want to throw an error with more context than just the property name

Reply:
The error thrown reads `'${oldName}' is deprecated and will be removed.`

It's brittle to test for a literal string though, so the match only looks for the property name

Reply:
ah that's fair! thanks for the explanation 💯